src/wp-includes/class-wp-duotone.php

```diff
@@ -814,7 +814,7 @@ private static function get_svg_definitions( $sources ) {
 	 * @return string The CSS for global styles.
 	 */
 	private static function get_global_styles_presets( $sources ) {
-		$css = 'body{';
```

Comment:
Might be worth adding `@since` comment up there with the change?

Reply:
Oh good spotting. Thanks! I'll fix.